Dockerfile

```diff
@@ -1,16 +1,5 @@
 # Dockerfile for git-bridge
 
+FROM maven:3-jdk-8
 
```

Comment:
I found that the container would crash pretty early with a `make not found` error. 

Adding this line:

```
RUN apt-get update && apt-get install -y git make curl
```
... solved the issue. I'll follow up on this on an internal branch before merging to master.

Reply:
What are you using make for?